api/iode.h

```diff
@@ -126,9 +126,9 @@
 #define KNB(kdb)     ((kdb)->k_nb)
 #define KOBJS(kdb)   ((kdb)->k_objs)
 
-#define KONAME(kdb, pos)    ((kdb)->k_objs[pos].o_name)
-#define KSOVAL(kdb, pos)    ((kdb)->k_objs[pos].o_val)
-#define KGOVAL(kdb, pos)    (SW_getptr((kdb)->k_objs[pos].o_val))
+#define KONAME(kdb, pos)    ((kdb)->k_objs[pos].o_name)                 // name of the object
```

Comment:
@jmpplan Could you tell me if the comments I've added for `KONAME`, `KSOVAL` and `KGOVAL` are correct, please?